docs/content/getting-started/setup-openfga.mdx

```diff
@@ -108,52 +108,41 @@ networks:
   openfga:
 
 services:
-  db:
+  postgres:
     image: postgres:14
+    container_name: postgres
     networks:
       - openfga
+    ports:
+      - "5432:5432"
     environment:
       - POSTGRES_USER=postgres
       - POSTGRES_PASSWORD=password
-      - POSTGRES_DB=openfga
     healthcheck:
       test: [ "CMD-SHELL", "pg_isready -U postgres" ]
       interval: 5s
       timeout: 5s
       retries: 5
 
-  pause:
-    image: ubuntu
-    command: sleep 5
-    depends_on:
-      db:
-        condition: service_started
-
   migrate:
-    image: openfga/openfga
     depends_on:
-      pause:
-        condition: service_completed_successfully
-    command: |
-      migrate
-    environment: 
-      - OPENFGA_DATASTORE_ENGINE=postgres
-      - OPENFGA_DATASTORE_URI=postgres://postgres:password@db:5432/openfga
+      postgres:
+        condition: service_healthy
+    image: openfga/openfga:latest
+    container_name: migrate
+    command: migrate --datastore-engine postgres --datastore-uri 'postgres://postgres:password@postgres:5432/postgres?sslmode=disable'
```

Comment:
Can we keep the references in the env variables instead of in the command? docker-compose users are more likely to want to configure through that

Reply:
I can do that.  The only question I have is whether we want to update https://github.com/openfga/openfga/blob/main/docker-compose.yaml to use that environment variables as well?  We want to minimize the delta between the two as much as possible

@craigpastro ?

Reply:
That sounds fine to me. I prefer env vars in any case.

Reply:
done

Reply:
Updated https://github.com/openfga/openfga/pull/526 to be more inline with this change